packages/metadata/src/metadata-service.ts

```diff
@@ -45,10 +45,10 @@ function getActionName(name: Path): Path {
 
 /**
  *
- * @param metadata
- * @param actionNames
- * @param namespaces
- * @param metadataElements
+ * @param metadata metadata element map
+ * @param actionNames string map
+ * @param namespaces set of namespace
```

Comment:
We can already see that the parameter has to be a set from type definitions, there is no need to repeat it in the description.

Reply:
updated